src/access/AccessControl.cpp

```diff
@@ -96,6 +99,8 @@ CHIP_ERROR AccessControl::Check(const SubjectDescriptor & subjectDescriptor, con
     {
         AuthMode authMode = AuthMode::kNone;
         ReturnErrorOnFailure(entry.GetAuthMode(authMode));
+        // Operational PASE not supported for v1.0.
+        VerifyOrReturnError(authMode == AuthMode::kCase || authMode == AuthMode::kGroup, CHIP_ERROR_INCORRECT_STATE);
```

Comment:
kGroup is valid. Why is kGroup an incorrect state at this point?

Reply:
It's OK, that line is doing:
verify that authmode is case or group, else return error.

Reply:
Got fooled again by reverse logic macro steering my eyes

Reply:
:-)